site/src/main/java/uk/nhs/digital/ps/components/DatasetComponent.java

```diff
@@ -22,7 +22,7 @@ public void doBeforeRender(final HstRequest request, final HstResponse response)
         super.doBeforeRender(request, response);
         final HstRequestContext ctx = request.getRequestContext();
         Dataset dataset = (Dataset) ctx.getContentBean();
-        Publication publication = getParentPublication(ctx, dataset);
+        Publication publication = dataset.getParentPublication(ctx);
```

Comment:
sugestion: pass "rootFolder" (`ctx.getSiteContentBaseBean()`) instead of ctx.

Reply:
I'm changing this again for RPS-170 so will merge this change into that and submit pull requests together